### PR TITLE
Remove default values from Larva tests; use 5s timeout instead of 20s…

### PR DESCRIPTION
**Pull request by jkosternl**
…; improve Javadoc & code; enable ActiveMQ console

Just some small improvements

### CODE REVIEW

File: test/src/main/configurations/JMS/ConfigurationEsbJmsListenerSender.xml [hunk 1]
```diff
@@ -13,9 +13,9 @@
 				className="org.frankframework.pipes.SenderPipe">
 				<sender className="org.frankframework.extensions.esb.EsbJmsSender"
 					destinationName="${tibco.rr_queue}" jmsRealm="qcf_tibco_esb_rr"
-					messageProtocol="RR">
+					messageProtocol="RR" timeOut="5000">
```
Comment (tnleeuw) [inline test/src/main/configurations/JMS/ConfigurationEsbJmsListenerSender.xml:16]: I hope this more limited timeout is not going to cause tests to fail when server is running slow.

Timeouts are only a problem when the reply is not going to come at all, it's not an issue when the reply will come. But a timeout that's too restrictive can be an issue when the server is not as fast for any reason.

Comment (jkosternl) [reply test/src/main/configurations/JMS/ConfigurationEsbJmsListenerSender.xml:16]: The default of 20 secs is very long. The ActiveMQ is already running fine, when it reaches this point of the code, so I figured after 5 secs, the response is already given and it is already long enough. So, it indeed just helps to fail fast when we made a mistake.

**Review by tnleeuw — APPROVED** [review]

File: test/src/main/configurations/JMS/ConfigurationEsbJmsListenerSender.xml (hunk 1 above)
Comment (nielsm5) [reply test/src/main/configurations/JMS/ConfigurationEsbJmsListenerSender.xml:16]: Don't we have a timeout (lowercase o) property? If not, we need to sort that out soon...

Comment (jkosternl) [reply test/src/main/configurations/JMS/ConfigurationEsbJmsListenerSender.xml:16]: I thought I tried with `timeout` firstly, and that failed. Than I looked it up and used `timeOut` and it works. I'll recheck.

Comment (nielsm5) [reply test/src/main/configurations/JMS/ConfigurationEsbJmsListenerSender.xml:16]: I found this: https://github.com/frankframework/frankframework/issues/5491
Perhaps we can pick this up soon 😄

Comment (jkosternl) [reply test/src/main/configurations/JMS/ConfigurationEsbJmsListenerSender.xml:16]: Ah.. valid point! 👍🏻

**Review by nielsm5 — APPROVED** [review]

**Pull request merged**